hnn_core/tests/test_parallel_backends.py

```diff
@@ -144,46 +146,46 @@ def test_run_mpibackend_oversubscribed(self, run_hnn_core_fixture):
             assert backend.n_procs == oversubscribed
             simulate_dipole(net)
 
-    @pytest.mark.parametrize("backend", ['mpi', 'joblib'])
-    def test_compare_hnn_core(self, run_hnn_core_fixture, backend, n_jobs=1):
-        """Test hnn-core does not break."""
-        # small snippet of data on data branch for now. To be deleted
-        # later. Data branch should have only commit so it does not
-        # pollute the history.
-        data_url = ('https://raw.githubusercontent.com/jonescompneurolab/'
-                    'hnn-core/test_data/dpl.txt')
-        if not op.exists('dpl.txt'):
-            _fetch_file(data_url, 'dpl.txt')
-        dpl_master = loadtxt('dpl.txt')
-
-        dpls, net = run_hnn_core_fixture(backend=backend)
-        dpl = dpls[0].smooth(30).scale(3000)
-
-        # write the dipole to a file and compare
-        fname = './dpl2.txt'
-        dpl.write(fname)
-
-        dpl_pr = loadtxt(fname)
-        assert_array_equal(dpl_pr[:, 2], dpl_master[:, 2])  # L2
-        assert_array_equal(dpl_pr[:, 3], dpl_master[:, 3])  # L5
-
-        # Test spike type counts
-        spike_type_counts = {}
-        for spike_gid in net.cell_response.spike_gids[0]:
-            if net.gid_to_type(spike_gid) not in spike_type_counts:
-                spike_type_counts[net.gid_to_type(spike_gid)] = 0
-            else:
-                spike_type_counts[net.gid_to_type(spike_gid)] += 1
-        assert 'common' not in spike_type_counts
-        assert 'exgauss' not in spike_type_counts
-        assert 'extpois' not in spike_type_counts
-        assert spike_type_counts == {'evprox1': 269,
-                                     'L2_basket': 54,
-                                     'L2_pyramidal': 113,
-                                     'L5_pyramidal': 395,
-                                     'L5_basket': 85,
-                                     'evdist1': 234,
-                                     'evprox2': 269}
+    # @pytest.mark.parametrize("backend", ['mpi', 'joblib'])
```

Comment:
make another pull request to the data branch with a new ground-truth. We'll merge the two PRs simultaneously.